wallet/src/main/scala/org/bitcoins/wallet/internal/UtxoHandling.scala

```diff
@@ -87,68 +88,95 @@ private[wallet] trait UtxoHandling extends WalletLogger {
     }
   }
 
-  protected def updateUtxoConfirmedState(
+  private[wallet] def updateUtxoConfirmedState(
       txo: SpendingInfoDb): Future[Option[SpendingInfoDb]] = {
     updateUtxoConfirmedStates(Vector(txo)).map(_.headOption)
   }
 
-  protected def updateUtxoConfirmedStates(
+  private[wallet] def getDbsByRelevantBlock(
```

Comment:
scaladoc

Reply:
Also what is the case for the key `None` in the Map? Also i'm unsure where the 'revelevant' part comes into play here? Shouldn't we should call it `groupSpendingInfoByBlock()`?

Reply:
> Also what is the case for the key None in the Map

mempool txs

>  Also i'm unsure where the 'revelevant' part comes into play here? Shouldn't we should call it groupSpendingInfoByBlock()

I used relevant because there are 2 txs "by block" could refer to, the receving, and the spending tx.

Reply:
Ok, the scaladoc really helps out. One last nit: Add the mempool case to the scaladoc too.

Reply:
ping